modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-filtering-ebpf-rule_{context}"]
+= Filtering eBPF flow data using a global rule
+You can configure the `FlowCollector` to filter eBPF flows using a global rule as a method to control the flow of packets cached in eBPF's flows table based on certain configuration.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for *Network Observability*, select *Flow Collector*.
+. Select *cluster* then select the *YAML* tab.
+. Configure the `FlowCollector` custom resource. A few sample configurations are as follows:
++
+
+[%collapsible]
+.Filter Kubernetes service traffic to specific POD IP endpoint
+====
+[source, yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  namespace: netobserv
+  deploymentModel: Direct
+  agent:
+    type: eBPF
+    ebpf:
+      flowFilter:
+        action: Accept  <1>
+        cidr: 172.210.150.1/24 <2>
+        protocol: SCTP
+        direction: Ingress
+        destPortRange: 80-100
+        peerIP: 10.10.10.10
+        enable: true    <3>
+----
+<1> The required `action` parameter describes the action to be taken for the flow filter rule. Possible values are `Accept` or `Reject`. 
```

Comment:
```suggestion
<1> The required `action` parameter describes the action taken by the flow filter rule. Possible values are `Accept` or `Reject`. 
```
The [Red Hat style guide](https://redhat-documentation.github.io/supplementary-style-guide/) cautions against future tense. Could this rewrite work?